templates/server/src/main/resources/archetype-resources/core/src/main/resources/db/type/hana/V0002__Create_RevInfo.sql

```diff
@@ -0,0 +1,7 @@
+-- *** RevInfo (Commit log for envers audit trail) ***
+CREATE COLUMN TABLE RevInfo(
+  id          BIGINT NOT NULL GENERATED BY DEFAULT AS IDENTITY (START WITH 1),
+  "timestamp" BIGINT NOT NULL,
```

Comment:
Unquoted columns will be upper-cased. So for consistency reasons you should name this column "TIMESTAMP".

Reply:
The reason here is that we had to quote this property also in the JPA annotation as this is a reserved keyword in Oracle. 
https://github.com/oasp/oasp4j/blob/develop/modules/jpa-basic/src/main/java/io/oasp/module/jpa/dataaccess/api/AdvancedRevisionEntity.java#L41

For some reasons (case-sensitivity) in DDL for most databases this property then also has to be quoted in the same way. To keep this consistent we do it the same way for the DDLs of all database types.

Reply:
BTW: Using "TIMESTAMP" in AdvancedRevisionEntity.java would indeed have been the better option for 2.6.x. Changing this again? Or leave it as is?